### DISCUSSION TITLE
NPI is missing in the In-Network Schema, yet in the example

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#14] [Q&A] [answered]
Within the In-Network Rate schema, I am not seeing the NPI included in the Negotiated Rate Details Object array.  It is my understanding that the NPI, TIN and Service Code are required.  I see the NPI in the example but not the schema.  
Shouldn't the NPI be in the schema?

**Comment by ghost** [comment]
Yes, it appears they dropped npi in the schema and using the key name of providers  as an array of provider identification numbers (NPI). This can be a list of type 1 and type 2 NPIs per the schema documentation.

But in the examples you see a reference to captivation using "npi" as key name
"providers": [{
        "tin": "",
        "service_code": "01",
        "npi": ["6666666666", "7777777777", "8888888888", "9999999999"]
      },{
        "tin": "",
        "service_code": "01",
        "npi": ["1111111111", "2222222222", "3333333333", "4444444444"]
      }]

while it is dropped and just using a array after the "provider" key value in the bundled it is
"providers": ["1111111111", "2222222222", "3333333333", "4444444444", "5555555555"],
      "tin": "11-1111111",

and fee for service
"providers": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
      "tin": "11-1111111",

**Reply by rgouldkc** [reply]
As a point of clarification, the documentation describes providers as an array of individual (type 1) provider identification numbers (NPI). The response above indicates it can be type 1 or type 2. In our case we have contracts with provider groups with a type 2 NPI which covers situations where an individual practitioner may not be contracted thus the claims would use the groups contracted rate. Based on that we would need to include at least some type 2 NPIs.

**Comment by tdfow** [comment]
So is that indicating the schema is accurate and the example needs updating to align?

**Reply by shaselton-usds** [reply]
Hi @tdfow -- here is the documentation requiring the NPI: https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object

It looks like the supporting example matching the schema. Are you seeing something different?

**Comment by tdfow** [comment, marked as answer]
The schema shows the NPI being reported as providers but in the capitation example it has an npi reference, grouped by providers--which doesn't match the schema--as there isn't an npi in schema.   Assume capitation example is wrong???

**Reply by shaselton-usds** [reply]
I see! Thanks. 

Yeah, the schema will have to be updated to support the arrangement/relationships in a capitation arrangement.

**Comment by ryan-thomas-peterson** [comment]
For many arrangements of facility based contracts rates are determined though Tax IDs and may not be differentiated based on entity type 2 NPIs,  In these arrangements may the provider NPI be omitted?

**Reply by shaselton-usds** [reply]
Documentation was updated to support [type 2 NPIs.](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#providers-object)
